main.py

```diff
@@ -37,20 +38,33 @@ def home(request: Request):
 
 
 @app.get("/classifications")
+# This is the get request for the classification form
 def create_classify(request: Request):
     return templates.TemplateResponse(
         "classification_select.html",
-        {"request": request, "images": list_images(), "models": Configuration.models},
+        {
+            "request": request,
+            "images": list_images(),
+            "models": Configuration.models},
     )
 
 
 @app.post("/classifications")
 async def request_classification(request: Request):
     form = ClassificationForm(request)
     await form.load_data()
+    # We retrieve the image id and model id from the form
     image_id = form.image_id
     model_id = form.model_id
+
+    # We classify the image using the model
     classification_scores = classify_image(model_id=model_id, img_id=image_id)
+
+    # We save the classification scores to the classification_scores.json file
+    with open('app/static/classification_scores.json', 'w') as f:
+        json.dump(classification_scores, f)
```

Comment:
What happens if multiple users ask for a classification job? The file would be overwritten. Isn't it best to return the results directly? Or, if you reuse them later for other things, another way to do it would be to store the results with a unique name and clean up after the response is returned to the user (or after a while anyway).